### PR TITLE
E2E: Fix for failing tests on nickname spec

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix for failing tests on nickname spec

#### Ticket Link
none, failing on Cypress daily test

### CODE REVIEW

File: e2e/cypress/integration/account_settings/general/nickname_spec.js [hunk 1]
```diff
@@ -111,4 +115,8 @@ describe('Account Settings > Sidebar > General', () => {
         cy.get('#nicknameDesc').should('be.visible').should('contain', "Click 'Edit' to add a nickname");
         cy.get('#accountSettingsHeader > .close').should('be.visible').click();
     });
+
+    function getInnerText(el) {
+        return el[0].innerText.replace(/\n/g, '').replace(/\s/g, ' ');
```
Comment (saturninoabril) [inline e2e/cypress/integration/account_settings/general/nickname_spec.js:120]: A quick hack to strip out unnecessary characters.

**Comment by saturninoabril** [comment]
Thanks for running the test.  Let me verify again.

**Comment by srkgupta** [comment]
Yeah I get the same error as pointed by @josephbaylon

**Comment by saturninoabril** [comment]
It's not related but I'll check.  Note that such code was ran daily and is passing - https://mm-cypress-report.s3.amazonaws.com/1588744149267/mochawesome.html

**Comment by saturninoabril** [comment]
Ran on my local and it's consistently passing (same result as the test server)
Screen Shot 2020-05-06 at 5 19 49 PM

**Pull request merged**